### PR TITLE
Implement BLOB, fix issues in PR #7, forward pagination, add emojis, implement cloud translation, and sentiment analysis

### PR DESCRIPTION
**Pull request by tonypan2000**

### CODE REVIEW

**Comment by tonypan2000** [comment]
I was working on another branch earlier today and had to merge the changes from Development back to that branch, so please ignore commit 5d895ea as it is identical to PR #7.

**Comment by tonypan2000** [comment]
I implemented a basic pagination that allows users to click "Next Page" link at the very bottom of the comment section to view the next n previous comments. The server side uses a cursor string to track the last comment being displayed. It currently does not allow for going back to the previous page, nor does it have page number indexing. Those features need more than just a cursor. issue #11

**Comment by tonypan2000** [comment]
Lets users add their mood to comment

**Comment by tonypan2000** [comment]
Cloud translation has been implemented for the comment section. The page does select the proper language and sends it to server as a query string. The server can translate Spanish ok, but the web page cannot display languages properly.

**Comment by tonypan2000** [comment]
Cloud Translation is working

**Comment by tonypan2000** [comment]
Now each comment also displays the sentiment score representing how positive or negative it is.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -65,15 +68,17 @@
     String content;
     String imageUrl;
     String mood;
+    float sentiment;
     String cursor;
 
-    private Comment(long id, String name, Long timestamp, String content, String imageUrl, String mood, String cursor) {
+    private Comment(long id, String name, Long timestamp, String content, String imageUrl, String mood, float sentiment, String cursor) {
```
Comment (jessrenteria) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:74]: Nit: try to adhere to having less than [100](https://google.github.io/styleguide/javaguide.html#s4.4-column-limit) characters per line.

This is much more important in a codebase with many people contributing / reading, and if you have a sufficiently large monitor it's really up to you. But right now I can't see all of it in my code review and have to scroll sideways which hurts readability.

**Pull request merged**